src/MFnTransform.inl

```diff
@@ -0,0 +1,507 @@
+    .value("kScaleMaxX", MFnTransform::kScaleMaxX)
+    .value("kScaleMinY", MFnTransform::kScaleMinY)
+    .value("kScaleMaxY", MFnTransform::kScaleMaxY)
+    .value("kScaleMinZ", MFnTransform::kScaleMinZ)
+    .value("kScaleMaxZ", MFnTransform::kScaleMaxZ)
+
+    .value("kShearMinXY", MFnTransform::kShearMinXY)
+    .value("kShearMaxXY", MFnTransform::kShearMaxXY)
+    .value("kShearMinXZ", MFnTransform::kShearMinXZ)
+    .value("kShearMaxXZ", MFnTransform::kShearMaxXZ)
+    .value("kShearMinYZ", MFnTransform::kShearMinYZ)
+    .value("kShearMaxYZ", MFnTransform::kShearMaxYZ)
+
+    .value("kRotateMinX", MFnTransform::kRotateMinX)
+    .value("kRotateMaxX", MFnTransform::kRotateMaxX)
+    .value("kRotateMinY", MFnTransform::kRotateMinY)
+    .value("kRotateMaxY", MFnTransform::kRotateMaxY)
+    .value("kRotateMinZ", MFnTransform::kRotateMinZ)
+    .value("kRotateMaxZ", MFnTransform::kRotateMaxZ)
+
+    .value("kTranslateMinX", MFnTransform::kTranslateMinX)
+    .value("kTranslateMaxX", MFnTransform::kTranslateMaxX)
+    .value("kTranslateMinY", MFnTransform::kTranslateMinY)
+    .value("kTranslateMaxY", MFnTransform::kTranslateMaxY)
+    .value("kTranslateMinZ", MFnTransform::kTranslateMinZ)
+    .value("kTranslateMaxZ", MFnTransform::kTranslateMaxZ)
+    .export_values();
+
+FnTransform
+    .def(py::init<>())
+
+    .def(py::init([](MObject& object) {
+        MStatus status;
+        auto result = std::unique_ptr<MFnTransform>(new MFnTransform(object, &status));
+
+        if (!status) {
+            throw std::runtime_error(
+                "Invalid parameter passed for object - "
+                "not a Transform Node, "
+                "Node does not exist or "
+                "no valid pointer to Node"
+            );
+        }
+
+        return result;
+    }), py::arg("object"))
+
+    .def(py::init([](MDagPath& dagPath) {
+        MStatus status;
+        auto result = std::unique_ptr<MFnTransform>(new MFnTransform(dagPath, &status));
+
+        if (!status) {
+            throw std::runtime_error(
+                "Invalid parameter passed for object - "
+                "not a Transform Node, "
+                "Node does not exist or "
+                "no valid pointer to Node"
+            );
+        }
+
+        return result;
+    }), py::arg("dagPath"))
+
+    .def("clearRestPosition", [](MFnTransform & self) {
+        MStatus status = self.clearRestPosition();
+
+        if (!status) {
+            throw std::logic_error(status.errorString().asChar());
+        }
+    }, _doc_FnTransform_clearRestPosition)
+
+    .def("create", [](MFnTransform & self, MObject parent = MObject::kNullObj) -> MObject {
+        MStatus status;
+        MObject transform = self.create(parent, &status);
+
+        if (!status) {
+            throw std::runtime_error(status.errorString().asChar());
+        }
+
+        return transform;
+    }, py::arg("parent") = MObject::kNullObj, _doc_FnTransform_create)
```

Comment:
In order to fix the stubs generation you'll need to use `py::arg_v` instead of `py::arg`.
This is because the `parent` argument has a default value that doesn't map well to python.
`py::arg_v` lets you provide the default value as well as the string that should be used in the stubs for that default value.

There's a bit more information on that [here](https://github.com/mottosso/cmdc/blob/master/.github/contributing.md#ensuring-good-stubs-generation) as well as links to pybind's documentation that goes a bit more in depth.


Here's what this line should be:
```suggestion
    }, py::arg_v("parent", MObject::kNullObj, "Object.kNullObj"), _doc_FnTransform_create)
```